src/test/junit/CreateNewCampaign.java

```diff
@@ -0,0 +1,57 @@
+package test.junit;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import pages.*;
+
+/**
+ * Created by Mario Perez on 6/16/2015.
+ */
+public class CreateNewCampaign {
+
+    private LoginPage loginPage;
+    private MainApp mainApp;
+
+    private AppBody appCampaign;
+
+    @Before
+    public void setUp() {
+        loginPage = new LoginPage();
+        String email = "mario.cbba@salesforce.com";
+        String password = "mario123";
+        String userName = "Mario Perez";
+
+        mainApp = loginPage.loginAs(email,password,userName);
```

Comment:
Create a loginAsPrimaryUser method and read the email, password from a properties file